template/$PROJECT_NAME$/test/$PROJECT_NAME$/account/user_token_service_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule <%= @project_name_camel_case %>.Account.UserTokenServiceTest do
+  use <%= @project_name_camel_case %>.DataCase
+
+  alias <%= @project_name_camel_case %>.Account.UserTokenService
+
+  describe "generate_token/2" do
+    test "returns a token if the email is associated with an email" do
+      user = insert(:user)
+      {:ok, token} = UserTokenService.generate_token(user.email, :email_confirmation)
+      assert token.user.id == user.id
+    end
+
+    test "returns error tuple when no user exists with that email" do
```

Comment:
You might want `returns an error tuple` for consistency with the next test. That or update the next test to match the others in this file